src/journey-maps/angular/services/map/events/map-selection-event.ts

```diff
@@ -52,9 +51,9 @@ export class SbbMapSelectionEvent {
     this._subscription?.unsubscribe();
   }
 
-  toggleSelection(eventData: SbbFeaturesClickEventData): void {
```

Comment:
Be aware that this is technically a breaking change. Try to avoid breaking changes or document them in the PR message (which requires a major version release).

Reply:
This service - and also all of our other services - is an internal service and mustn't be used by our users. Therefore I consider this an internal refactoring and no breaking change. The `@Output` event which will be received by the user has not changed.

Do you agree with that?

Reply:
Hm, we lack a clear distinction between public and private API for these internal services.
It's fine for now.